universal/components/ContentList/component.js

```diff
@@ -47,35 +46,19 @@ function renderList ({
   )
 }
 
-const defaultList = [
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item'
-]
-
-function ContentList ({
-  hasTitle = true,
+const ContentList = ({
+  customClass,
+  hasTitle,
   title,
-  bulletList = true,
-  items = defaultList,
-  itemWidth = 'half'
-}: ContentListProps) {
+  bulletList,
+  items,
+  itemWidth
+}: ContentListProps) => {
   return (
-    <SectionFullWidth>
-      <SectionContainer>
-        {renderTitle({hasTitle, title})}
-        {renderList({bulletList, items, itemWidth})}
-      </SectionContainer>
-    </SectionFullWidth>
+    <div className={customClass}>
```

Comment:
Can we have `className={customClass}` only render into that div if that prop has something input for it?